guides/common/attributes-titles.adoc

```diff
@@ -18,7 +18,7 @@
 :ManagingHostsDocTitle: Managing Hosts
 :ManagingOrganizationsLocationsDocTitle: Managing Organizations and Locations in {ProjectName}
 :ManagingSecurityDocTitle: Managing Security Compliance
-:MonitoringDocTitle: Monitoring {ProjectName}
+:MonitoringDocTitle: Monitoring {Project} Performance
```

Comment:
This should also be updated in `web/content/js/versions.js`

Reply:
Thank you!